src/_igraph/graphobject.c

```diff
@@ -13895,6 +13895,146 @@ PyObject *igraphmodule_Graph_random_walk(igraphmodule_GraphObject * self,
   }
 }
 
+
+/**********************************************************************
+ * Other methods                                                      *
+ **********************************************************************/
+
+/**
+ * Voronoi clustering
+ */
+PyObject *igraphmodule_Graph_community_voronoi(igraphmodule_GraphObject *self,
+                                               PyObject *args, PyObject *kwds) {
+    static char *kwlist[] = {"lengths", "weights", "mode", "radius", NULL};
+    PyObject *lengths_o = Py_None, *weights_o = Py_None;
+    PyObject *mode_o = Py_None;
+    PyObject *radius_o = Py_None;
+    igraph_vector_t *lengths_v = 0;
+    igraph_vector_t *weights_v = 0;
```

Comment:
I know that this codebase frequently uses `0` for null pointers, but let's aim to be clear with newly added code: `NULL` should be used for pointers, `0` for numbers, and `false` for Booleans.